### PR TITLE
v0.78.0 — Card component section child styles.

### PR DESCRIPTION
**Pull request by DamianMullins**
### Added
- Card component section child styles.

card-section--collapsible

### CODE REVIEW

File: src/scss/components/_cards.scss [hunk 1]
```diff
@@ -62,5 +66,68 @@ $card-borderColor               : $grey--lightest !default;
...
+
+            .c-card-section--collapsed {
+                height: 50px;
+                overflow: hidden;
+
+                &.c-card-section {
+                    margin-bottom: -#{spacing(x2)};
+                }
+
+                p {
```
Comment (sonnyprince) [inline src/scss/components/_cards.scss]: Should add a class and target that instead, something like `.c-card-section-text` - just incase the the `p` changes in the future :)

Comment (DamianMullins) [reply src/scss/components/_cards.scss]: The thing is, the text comes from the back-end somewhere so we can't easily add a class to the content.

Comment (sonnyprince) [reply src/scss/components/_cards.scss]: ahh - bummer. No worries then 😄

Comment (DamianMullins) [reply src/scss/components/_cards.scss]: I'll keep your comment in mind though as there may be an opportunity to fix that text from within the menu vertical.

**Pull request merged**